### DISCUSSION TITLE
Translate to English not working consistently

### DISCUSSION THREAD

**Discussion by Veneration1** [ggml-org/whisper.cpp#1167] [Q&A]
Hi Everyone,
I've been trying to get whisper to translate Spanish to English with poor results.

Originally I was using "--language en", and it works about 25% of the time. The times it doesn't work for the time stamps it says something like [SPANSH] or [FOREIGN] or [Silver speaking Spanish] (whatever that means).
I also tried "--translate." Then I tried with both --language en and --translate and any variation thereof, but it still doesn't translate most of the time.
Any suggestions? 

Thank you!

**Comment by bobqianic** [comment]
Interesting. Could you share the transcription output and the parameters you used? It would also be great if you could provide the audio samples.
